src/block/blog-posts/index.php

```diff
@@ -140,6 +140,11 @@ function stackable_render_blog_posts_block( $attributes, $content ) {
 
 		$recent_posts = wp_get_recent_posts( $post_query );
 
+		// Manually slice the array based on the number of posts per page.
+		if ( sizeof( $recent_posts ) > $post_query['numberposts'] ) {
```

Comment:
Please use `count` instead of sizeof. Also, `wp_get_recent_posts` can return `false` if no posts are found, you'll need to check for this.